### PR TITLE
Increase cmake minimum required [16436]

### PR DESCRIPTION
**Pull request by richiware**
Increase version to one is being testing our CI

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/81/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/76/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Linux/82/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Manual%20Windows/77/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@
 ###############################################################################
 # CMake build rules for Micro CDR
 ###############################################################################
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
```
Comment (Ryanf55) [inline CMakeLists.txt:18]: CI is not updated to have this new version. Linux jobs are failing due to it. 
http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Linux/1232/console

```
[Micro-CDR Nightly Master Linux] $ /home/jenkins/tools/hudson.plugins.cmake.CmakeTool/3.10.0/bin/cmake --version
cmake version 3.10.0
...
CMake Error at CMakeLists.txt:18 (cmake_minimum_required):
  CMake 3.15 or higher is required.  You are running version 3.10.0


-- Configuring incomplete, errors occurred!
```

Comment (pablogs9) [reply CMakeLists.txt:18]: @richiware

**Pull request merged**